llvm/test/tools/llvm-foreach/llvm-foreach-lin.ll

```diff
@@ -4,15 +4,22 @@
 ; RUN: echo 'Content of second file' > %t2.tgt
 ; RUN: echo "%t1.tgt" > %t.list
 ; RUN: echo "%t2.tgt" >> %t.list
-; RUN: llvm-foreach --in-replace="{}" --in-file-list=%t.list -- echo "{}" > %t.res
+; RUN: llvm-foreach --jobs=2 --in-replace="{}" --in-file-list=%t.list -- echo "{}" > %t.res
 ; RUN: FileCheck < %t.res %s
-; CHECK: [[FIRST:.+1.tgt]]
-; CHECK: [[SECOND:.+2.tgt]]
-;
+; CHECK-DAG: [[FIRST:.+1.tgt]]
+; CHECK-DAG: [[SECOND:.+2.tgt]]
+
+; RUN: llvm-foreach --in-replace="{}" --in-file-list=%t.list -- echo "{}" > %t.res
```

Comment:
We should also have `llvm-foreach --jobs=1` with `CHECK-ORDER` check-lines

Reply:
`--jobs=1` is the default argument value which is anyway set implicitly, I believe there is no need for explicit checking.